doc/source/structures/misc/config.rst

```diff
@@ -209,3 +213,17 @@ Configuration of kOS
     To make the change take effect you may have to
     stop, then restart the telnet server, as described above.
 
+.. attribute:: Config:DEBUGEACHOPCODE
+
+    :access: Get/Set
+    :type: boolean
+
+    Configures the ``DebugEachOpcode`` setting.
+
+    If true, each opcode that is executed by the CPU will be accompanied by 
+    an entry in the KSP log. This is a debugging tool for those who are very 
+    familiar with the inner workings of kOS and should rarely be used outside
+    the kOS dev team
```

Comment:
I'd also be sure to mention to people that it greatly lags the game by a lot, just to help discourage it's use.